pkgs/applications/science/biology/kent/default.nix

```diff
@@ -53,6 +53,7 @@ stdenv.mkDerivation rec {
 
     cp ../lib/x86_64/jkOwnLib.a $NIX_BUILD_TOP/lib
     cp ../lib/x86_64/jkweb.a $NIX_BUILD_TOP/lib
+    cp -r ../inc  $NIX_BUILD_TOP/
```

Comment:
Please avoid using NIX_BUILD_TOP

Reply:
Thanks for the review. What would be the proper way to handle it ?

Reply:
it is probably the best to use relative paths or cd before copying that the files are in the current directory